install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -15,6 +15,13 @@ spec:
       labels:
         name: strimzi-cluster-operator
         strimzi.io/kind: cluster-operator
+        com.company: Red_Hat
+        rht.prod_name: Red_Hat_Integration
+        rht.prod_ver: "7.8"
```

Comment:
I'm not completely sure what this should be. The Integration version will be probably 2021.Q4? Is this label mandatory?